backend/filejump/filejump.go

```diff
@@ -0,0 +1,1969 @@
+			Default:  "0",
+			Advanced: true,
+		}, {
+			Name:     "upload_cutoff",
+			Help:     "Cutoff for switching to multipart upload (>= 50 MiB).",
+			Default:  fs.SizeSuffix(defaultUploadCutoff),
+			Advanced: true,
+		}, {
+			Name:     "hard_delete",
+			Help:     "Delete files permanently instead of moving them to trash.",
+			Default:  false,
+			Advanced: true,
+		}, {
+			Name:     config.ConfigEncoding,
+			Help:     config.ConfigEncodingHelp,
+			Advanced: true,
+			Default: (encoder.Display |
+				encoder.EncodeInvalidUtf8 |
+				encoder.EncodeLeftSpace |
+				encoder.EncodeRightSpace),
+		}},
+	})
+}
+
+// Options defines the configuration for this backend
+type Options struct {
+	UploadCutoff fs.SizeSuffix        `config:"upload_cutoff"`
+	Enc          encoder.MultiEncoder `config:"encoding"`
+	AccessToken  string               `config:"access_token"`
+	WorkspaceID  string               `config:"workspace_id"`
+	HardDelete   bool                 `config:"hard_delete"`
+}
+
+// Fs represents a remote filejump server
+type Fs struct {
+	name     string
+	root     string
+	opt      Options
+	features *fs.Features
+	srv      *rest.Client
+	pacer    *fs.Pacer
+	dirCache *dircache.DirCache
+}
+
+// Object describes a filejump object
+type Object struct {
+	fs          *Fs
+	remote      string
+	hasMetaData bool
+	size        int64
+	modTime     time.Time
+	id          string
+}
+
+// Workspace describes a filejump workspace
+type Workspace struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
+// CallJSON is a generic function for API calls
+/*
+POST-Request Template:
+--------------------
+type Request struct {
+    Field1 string `json:"field1"`
+    Field2 int    `json:"field2"`
+}
+request := Request{
+    Field1: "value1",
+    Field2: 42,
+}
+
+type Response struct {
+    Status string `json:"status"`
+    Data   string `json:"data,omitempty"`
+}
+
+result, err := CallJSON[Response, Request](ctx, f, "POST", "/api/endpoint", nil, &request)
+if err != nil {
+    return err
+}
+
+GET-Request Template:
+-------------------
+type Response struct {
+    Status string `json:"status"`
+    Data   string `json:"data,omitempty"`
+}
+values := url.Values{}
+values.Set("param1", "value1")
+values.Set("param2", "value2")
+values.Set("EntryIds", fmt.Sprintf("[%s]", dir))
+values.Set("DeleteForever", strconv.FormatBool(true)))
+
+result, err := CallJSON[Response, struct{}](ctx, f, "GET", "/file-entries/delete", &values, nil)
+if err != nil {
+    return err
+}
+*/
+func CallJSON[T any, B any](ctx context.Context, f *Fs, method string, path string, params *url.Values, body *B) (*T, error) {
+	// Input parameters in one line
+	logMsg := fmt.Sprintf("CallJSON: %s %s", method, path)
+	if params != nil && len(*params) > 0 {
+		logMsg += fmt.Sprintf(" params=%s", params.Encode())
+	}
+	if body != nil {
+		bodyJSON, _ := json.Marshal(body)
+		logMsg += fmt.Sprintf(" body=%s", string(bodyJSON))
+	}
+	fs.Debugf(f, "%v", logMsg)
```

Comment:
Can you remove the debugging out of this function please.

You can see the transactions with `-vv --dump headers/bodies/requests/responses`